### PR TITLE
hc-added feature request template to repo

### LINKED ISSUE #12: [ADD] New Feature Issue Template
When you want to propose a new feature.

### PR DESCRIPTION
**Pull request by homiecoder**
Closes #12

Added feature issue template.

-hc

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Constraints and Assumptions
+
+_Call out any constraint/blockers and/or assumptions relevant for the development and use of this feature:_
+
+## Design Changes
+
+_Provide a high level description of the design highlighting the impacted OSC components, i.e.: REST API, data entities, SDKs, etc.:_
+
+</br>
```
Comment (kjaymiller) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: It's markdown... breaks are necessary
```suggestion
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
One minor change then you should be good.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (homiecoder) [reply .github/ISSUE_TEMPLATE/feature_request.md:17]: ok

**Comment by kjaymiller** [comment]
Cool go ahead and merge

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**